src/components/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name } = props;
+  return (
+    <div>
+      {name}
+    </div>
+  );
+};
+
+Button.defaultProps = {
+  name: null,
+};
+
+Button.propTypes = {
+  name: PropTypes.string,
```

Comment:
The `name` props should be `required` because the button would be useless without it, after all, we can't have an empty button. Even if you need an empty button, you can pass `null` yourself.
```suggestion
  name: PropTypes.string.isRequired,
```